### ISSUE TITLE
Does not show all songs in a Playlist

### ISSUE THREAD

**Issue by angelbarrera92** [An0nimia/deezloader#1]
Hi, thanks for this awesome work. 
When a invoke: 

```python
download_playlistdee("https://www.deezer.com/us/playlist/2249258602", output="/tmp/new_dance", check=False)
```
This only list 25 songs, but the playlist has 80 songs. Is the correct behaviour?
Thanks!

**Comment by An0nimia** [comment]
Problem has resolved with release 1.9. Thanks for the request :).

**Comment by angelbarrera92** [comment]
Seems to work @An0nimia
